test/mocks/config/mocks.h

```diff
@@ -94,6 +94,10 @@ class MockGrpcMux : public GrpcMux {
   MockGrpcMux();
   ~MockGrpcMux() override;
 
+  using GrpcMux::pause;
```

Comment:
why did you need this? Does marking MOCK_METHOD below override in 4th parameter (like below) fixes gcc build?

```
MOCK_METHOD(void, start, (), (override));
```